Android/app/src/main/java/com/didichuxing/doraemondemo/amap/AMapRouterFragment.kt

```diff
@@ -147,19 +147,27 @@ class AMapRouterFragment : CommBaseFragment() {
         //aMap.getUiSettings().setMyLocationButtonEnabled(true);设置默认定位按钮是否显示，非必需设置。
         // 设置为true表示启动显示定位蓝点，false表示隐藏定位蓝点并不进行定位，默认是false。
         mAmap.isMyLocationEnabled = true
-        //规划路径
-        mAMapNavi = AMapNavi.getInstance(activity?.application)
+        //确保调用SDK任何接口前先调用更新隐私合规updatePrivacyShow、updatePrivacyAgree两个接口并且参数值都为true，若未正确设置有崩溃风险
+        //官方文档：https://lbs.amap.com/api/android-navi-sdk/guide/create-project/configuration-considerations#t3
+        NaviSetting.updatePrivacyShow(context, true, true)
+        NaviSetting.updatePrivacyAgree(context, true)
+        try {
```

Comment:
- should not catch here, fix amap issues
- too generic exception caught

Reply:
done